.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Welcome to the OpenTracing Specification repo
+
+- Please search for existing issues in order to ensure we don't have duplicate bugs/feature requests.
+- Please be respectful and considerate of others when commenting on issues.
+- Please provide as much information as possible so we all understand the issue.
+- Please don't ask questions here. If you have any question head to our gitter chat https://gitter.im/opentracing/public
```

Comment:
As a GitHub hosted project, is "don't allow ask question through Issue" a good idea? I prefer we don't make this rule.

Reply:
That's fair.  I'll reword it to instead point people to try gitter first before creating an issue for a  question just so that issues would contain more of a specification-centric discussion and not necessarily the `how-to`'s of OpenTracing.

Reply:
Thinking more about it, I'll remove this line entirely.  Thanks!

Reply:
I rather liked the idea to direct people to gitter for how-to questions. But I don't think it should block this pr.